programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -1201,27 +1211,16 @@ static void on_event(const base_event *ev) {
                 struct add_or_edit_service_nrpt_req *edit_svc_req_data = calloc(1, sizeof(struct add_or_edit_service_nrpt_req));
                 edit_svc_req_data->hostnames = hostnamesToEdit;
                 edit_svc_req_data->dns_ip = get_dns_ip();
-
-                uv_async_t *ar = calloc(1, sizeof(uv_async_t));
-                ar->data = edit_svc_req_data;
-                uv_async_init(main_ziti_loop, ar, remove_and_add_nrpt_rules);
-                uv_async_send(ar);
+                ziti_tunnel_async_send(tunneler, remove_and_add_nrpt_rules, edit_svc_req_data);
```

Comment:
We should not need to use async calls from within `on_event`, since `on_event` is always called by the ziti_sdk which is on the loop thread.

Reply:
When I called the nrpt_rules scripts directly, ziti tunnel started crashing. So I used this ziti_tunnel_async_send in the on_event functions